doc/source/araalipolicy.rst

```diff
@@ -696,3 +696,4 @@ Subscribing to Perimeter Egress::
 Unsubscribing completely::
 
     ./araalictl api -unsubscribe-from-alert
+
```

Comment:
is this intentional? you can use tkdiff (or git difftool, or git diff) to self-review before submitting the pull request